src/renderer/components/duration/reactive-duration.tsx

```diff
@@ -34,6 +34,10 @@ function computeUpdateInterval(creationTimestampEpoch: number): number {
 }
 
 export const ReactiveDuration = observer(({ timestamp, compact = true }: ReactiveDurationProps) => {
+  if (!timestamp) {
+    return <>{"<unknown>"}</>;
```

Comment:
Nit: Why the react fragments? Is it because typing somewhere else, for a children-prop perhaps, requires JSX instead of string? It's better to have a proper type at source instead of having the complexity propagate :)

Reply:
Because react requires that functional components return react elements.

Reply:
I feel like there must be a better way. Another day :)